desktop/apps/auction/components/basic_checkbox/index.js

```diff
@@ -1,13 +1,13 @@
-import classNames from 'classnames'
+import PropTypes from 'prop-types'
 import React from 'react'
+import classNames from 'classnames'
```

Comment:
Natural sort takes into account cases, which often indicate different types; (`SomeClass` vs `someFunction`, for example.) 

Also! I know this is all super minor, I just tend to sort things whenever I come across them.